vividus-tests/src/main/resources/story/system/mobile_app/BrowserStackHealthCheckMobileAppSteps.story

```diff
@@ -6,3 +6,10 @@ Given I start mobile application with capabilities:
 When I close mobile application
 When I save BrowserStack network logs to SCENARIO variable `networkLogs`
 Then number of JSON elements from `${networkLogs}` by JSON path `$.log` is equal to 1
+
+Given I start mobile application with capabilities:
```

Comment:
no requirement id?

Reply:
Why it's a single scenario named `Verify BrowserStack integration`?
Looks too generic

Reply:
- correct, no requirement id
- renamed